src/services/SyncService.js

```diff
@@ -98,8 +98,8 @@ class SyncService {
 
 		// TODO: Only continue if a connection was made
 		this.connection = await connect
-
-		this.version = this.connection.docStateVersion
+		this.backend = new PollingBackend(this, this.connection)
+		this.version = this.connection.lastSavedVersion
```

Comment:
Was this reverted on purpose from https://github.com/nextcloud/text/pull/3842/files#diff-103df33bb085aaec0d6134982c312e96ecf58cf09335b9cf15964e17a45b4071 ?

Reply:
good catch... no that's left from the rebase!

Reply:
good catch... no that's left from the rebase!